client/my-sites/guided-transfer/style.scss

```diff
@@ -0,0 +1,20 @@
+.guided-transfer__success-hero-banner {
```

Comment:
I wasn't aware of those, thanks for catching me there :)

Rewrote this PR to use the existing thank-you pages in 76f089f